app/javascript/dashboard/routes/dashboard/settings/billing/Index.vue

```diff
@@ -18,6 +18,14 @@
           "
         />
       </div>
+      <div
+        v-tooltip.top="$t('BILLING_SETTINGS.COPY_CODE')"
+        class="stamp"
+        @click="copyToClipboard"
+      >
+        Use Code<br />
```

Comment:
```suggestion
        Use Code for 25%<br />
```

Reply:
Should it say "User code for 25% off" instead?